### PR TITLE
Create contributing.md

### LINKED ISSUE #295: New git flow
We now have a develop branch, which we will use to collect features before they are merged into master (and thus pulled onto the production server).

Before changes can be merged into master, they must be accompanied by an updated [CHANGELOG.md](https://github.com/developmentseed/scoreboard/blob/develop/CHANGELOG.md). 

The Changelog will describe the changes and collect them under a version corresponding to the types of updates that have been made. Versioning will follow MAJOR.MINOR.PATCH.

Once the PR is merged to master, two tasks must immediately follow:
- [ ] a git tag corresponding to the version in the Changelog should be applied to master
- [ ] the new master branch will be pulled onto production

A PR should not be merged into master unless the merger is prepared to execute the two above tasks.

cc @kamicut @LanesGood

### PR DESCRIPTION
**Pull request by LanesGood**
Proposes new contributing.md file. Closes #295

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# How to contribute to Scoreboard
+
+## Steps to contributing
+
+- Fork the repository
+- Create a branch for your changes
+- Make the changes you'd like to contribute. See the "types of contributions" list above to learn more about what we're looking for
```
Comment (kamicut) [inline contributing.md]: What are the types of contribution?

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- Submit a pull request
+
+Before changes can be merged into master, they _must_ be accompanied by an update to the [CHANGELOG.md](https://github.com/developmentseed/scoreboard/blob/develop/CHANGELOG.md). Please file all new changes under the "Unreleased" heading in CHANGELOG.md, under the appropriate subheading of "Added," "Changed" or "Fixed."
```
Comment (kamicut) [inline contributing.md]: Let's add here that we use https://keepachangelog.com/en/1.0.0/

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+The Changelog will describe the changes and collect them under a version corresponding to the types of updates that have been made. Versioning will follow MAJOR.MINOR.PATCH.
+
+Once the PR is merged to master, two tasks must immediately follow:
```
Comment (kamicut) [inline contributing.md]: This is something that we will do, no? the PR will be merged into develop, and we are going to add the git tag

File: contributing.md (hunk 1 above)
Comment (LanesGood) [reply contributing.md]: @kamicut this was copied from a similar file, I'll delete for now.

File: contributing.md (hunk 2 above)
Comment (LanesGood) [reply contributing.md]: @kamicut That's included in the Changelog itself... I don't think it's vital to link out from this page as well

File: contributing.md (hunk 1 above)
Comment (LanesGood) [inline contributing.md]:
```suggestion
- Make the changes you'd like to contribute.
```

**Pull request merged**